### ISSUE TITLE
Review and redevelop name matching code

### ISSUE THREAD

**Issue by M-Nicholls** [AtlasOfLivingAustralia/data-management#176]
maintainable rules
re-usable infrastructure
international collaboration

**Comment by nielsklazenga** [comment]
Can I be in on this, please?

**Comment by M-Nicholls** [comment]
Absolutely, but it's just placeholder at the moment, we probably won't get on to this until the second half of the year.  We need to settle the bugs out of the new BIE and names index first.
